deployment/bumpVersion.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Use ./bumpVersion.sh VERSION
+## Script upgrades the version numbers in the places necessary across the repo.
+## It will add the current and next version to appropiate locations.
+## Used for after a release to start work on next version.
+
+set -e -u -o pipefail
+
+if [  $# -eq 0  ]
+  then
+    echo "Version not specified. Usage ./bumpVersion.sh VERSION"
```

Comment:
it would be good if this also tells you the `CURRENT_VERSION`

Reply:
Good point, have made it print out the current version at the start either way.

Reply:
👍